about.md

```diff
@@ -20,6 +20,12 @@ Servo is written in [**Rust**](https://www.rust-lang.org/), taking advantage of
 
 Since its creation in 2012, Servo has contributed to W3C and WHATWG web standards, reporting specification issues and submitting new cross-browser automated tests, and core team members have co-edited new standards that have been adopted by other browsers. As a result, the Servo project helps drive the entire web platform forward, while building on a platform of reusable, modular technologies that implement web standards.
 
+## WPT pass rates
```

Comment:
I'd put this section after "Roadmap" at least, but maybe it's just me. A suggestion anyway.

Reply:
done in 449f674274161342ed408442520f5b4a60274228